src/main/java/com/epam/aidial/core/data/Publication.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.aidial.core.data;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import java.util.List;
+
+@Data
+@Accessors(chain = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Publication {
```

Comment:
Can you add javadocs to the fields? This class represent publication_create body. What if my request contain createdAt or status

Reply:
It is used as publication request, that's true. Nothing will happen because, they are reset. A bit tricky, but otherwise we need to create lots of dto classes for each type of request.

Reply:
What is not clear? I mean I can add java doc like url - is publication url, but does it really help?

Reply:
Added